openapi.yaml

```diff
@@ -83,108 +121,85 @@ parameters:
     type: string
     required: true
     description: '"Bearer [token]" where [token] is your OAuth2 access token'
-  pageNumber:
-    name: page[number]
-    in: query
-    type: integer
-    required: false
-    description: Page number of results
-    default: 1
-    minimum: 1
-  pageSize:
-    name: page[size]
-    in: query
-    type: integer
-    required: false
-    description: Number of results to return
-    default: 25
-    maximum: 500
-    minimum: 1
 definitions:
-  Meta:
-    properties:
-      totalResults:
-        type: integer
-        description: Total number of results
-        example: 10
-      totalPage:
-        type: integer
-        description: Total number of pages
-        example: 10
-      currentPageNumber:
-        type: integer
-        description: Page number of the returned results
-        example: 1
-      currentPageSize:
-        type: integer
-        description: Number of results per page
-        example: 25
   SelfLink:
     properties:
       self:
         type: string
         format: url
         description: Self-link of current resource
-  PaginationLinks:
-    properties:
-      first:
-        type: string
-        format: url
-        description: The first page of data
-      last:
-        type: string
-        format: url
-        description: The last page of data
-      prev:
-        type: string
-        format: url
-        description: The previous page of data
-      next:
-        type: string
-        format: url
-        description: The next page of data
-  PetResource:
+  TermResource:
     properties:
       id:
         type: string
-        description: A unique ID of pet
+        description: A 6-digit unique term code
       type:
         type: string
-        enum: [pet]
+        enum: [term]
       links:
         $ref: '#/definitions/SelfLink'
       attributes:
         properties:
-          name:
+          description:
+            type: string
+            description: Human readable academic term description
+            example: Winter 2019
+          season:
+            type: string
+            description: Season of the term
+            example: Winter
+            enum: ['Fall', 'Winter', 'Summer', 'Spring']
+          calendarYear:
+            type: string
+            description: A 4-digit calendar year
+            example: 2019
+          academicYear:
             type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
+            description: Academic year associated with this term. This is a 4-digit code
+            example: 1819
+          financialAidYear:
             type: string
-            description: Name of owner
-            example: Harry Potter
-          species:
+            description: Aid year associated with this term. This is a 4-digit code. For example, for Summer 2018, the value would be 1819
+            example: 1819
```

Comment:
Could you surround these examples in quotes? I'm pretty sure YAML parsers would interpret this as a number.